resources/definitions/strateo3d_IDEX420_duplicate.def.json

```diff
@@ -0,0 +1,131 @@
+{
+    "version": 2,
+    "name": "IDEX420 Duplicate",
```

Comment:
I'm not quite sure why you're adding a new printer definition here... It seems like a weird way to do this. The way it's setup now, also means that you won't even have shared profiles between the definitions.

Reply:
Hello nallath, I have no choice to add 2 other definitions of the machine since the working volume of the extruders are signifanctly different. Indeed, i don't have shared profiles, but, for now, except the time for updates which is a little more than for one machine (but, in VScode, copy-paste are quiete easy and fast and checking files is also easy), everything works propely and i can adjust temperature, speed, flow and other parameters for each configuration.

Reply:
You can have shared profiles, by using the "quality_definition". You can see how that works by looking at Ultimaker3 and Ultimaker3_extended. Even though they are differently sized, they do share the same profiles.

Reply:
Also; if you don't use the quality_definition trick, users also can't share profiles between the two machines. So it's not just about this being easy for you or not, but also if it's actually usable for people that use cura :P

Reply:
Hello nallath. I see that :
1) ultimaker3_extended inherits from ultimaker3 (.def.json files).
2) Quality profiles only apply on ultimaker3 (quality/ultimaker3/xxx.inst.cfg files)
So, if I understand, since 1) and 2), quality profiles also apply to ultimaker3_extended. Am I right?

Reply:
The reason that profiles are accepted by both is because the Ultimaker3_extended sets `"quality_definition": "ultimaker3"`

This way you don't have to copy all qualities multiple times and quality profiles created by users are usable by both.

Reply:
Hello nallath, I rewrite most of the files to be compliant to the ultimaker paradigm and architectur. It seems to work quite weel, however, I have a #2 on the machines names in Cura. Maybe I will need to uninstall both Cura 4.11 and 5.1 and flush everything to have the correct name.
But, whatever, your explanation was, indeed, a real time saving and, now, every parameter is very easy to understand and avoid multiple declaration and over-over-overrides...
Thanks a lot